pages/_document.js

```diff
@@ -36,7 +38,7 @@ export default class MyDocument extends Document {
         <Head>
           <meta
             name='viewport'
-            content='width=device-width,initial-scale=1'
+            content={`width=device-width,initial-scale=1 ${inNativeApp ? ',maximum-scale=1.0,user-scalable=0' : ''}`}
```

Comment:
ends up being `width=device-width,initial-scale=1 ,maximum-scale=1.0,user-scalable=0` or `width=device-width,initial-scale=1 `. Small detail and probably works but IMO the space should go or be after every coma: `width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=0`.

Reply:
yes! you're right. Going to fix it